src/main/java/mate/controller/drivers/AddDriverController.java

```diff
@@ -0,0 +1,34 @@
+package mate.controller.drivers;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.DriverService;
+
+public class AddDriverController extends HttpServlet {
+    private static final Injector injector
+            = Injector.getInstance("mate");
+    private DriverService driverService
+            = (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/drivers/addDriver")
+                .forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Driver driver = new Driver();
+        driver.setName(req.getParameter("name"));
+        driver.setLicenseNumber(req.getParameter("license_number"));
+        driverService.create(driver);
+        resp.sendRedirect(req.getContextPath() + "/addDriver");
```

Comment:
```suggestion
        resp.sendRedirect(req.getContextPath() + "/add");
```
fix other endpoints accordingly